ambry-frontend/src/test/java/com.github.ambry.frontend/AmbryBlobStorageServiceTest.java

```diff
@@ -142,8 +142,8 @@ public AmbryBlobStorageServiceTest() throws Exception {
     clusterMap.setPermanentMetricRegistry(metricRegistry);
     frontendConfig = new FrontendConfig(verifiableProperties);
     accountAndContainerInjector = new AccountAndContainerInjector(accountService, frontendMetrics, frontendConfig);
-    urlSigningService = new AmbryUrlSigningService(frontendConfig.frontendUrlSignerUploadEndpoint,
-        frontendConfig.frontendUrlSignerDownloadEndpoint, frontendConfig.frontendUrlSignerDefaultUrlTtlSecs,
+    urlSigningService = new AmbryUrlSigningService(frontendConfig.DEFAULT_ENDPOINT,
```

Comment:
if it is just for this, I think you can just send in some random strings here?

Reply:
Okay